qsimcirq/qsim_circuit.py

```diff
@@ -29,58 +29,59 @@ def _cirq_gate_kind(gate):
       + 'only 1- or 2-qubit gates are supported.')
   if isinstance(gate, cirq.ops.XPowGate):
     # cirq.rx also uses this path.
-    if gate.exponent == 1:
+    if gate.exponent == 1 and gate.global_shift == 0:
```

Comment:
To simplify this, we could instead add the following block at the start of this function:
```
if isinstance(gate, cirq.EigenGate) and gate.global_shift != 0:
  return NotImplementedError(f'qsim does not support global shift.
      Received gate with global shift of {gate.global_shift}.')
```
(`global_shift` is a property of the `EigenGate` class)

Reply:
I see - I misunderstood. This should be good to go, then.